### PR TITLE
Developing features

### PR DESCRIPTION
**Pull request by uimarshall**
This project entails the cloning of the Newsweek web using bootstrap. 
In this project I did the following:

- Layout the structure of the page
- Styled the page with the bootstrap framework with minimal css.
- Design for different screen sizes.

**After Review**

- Made sure that the text of my links always stays in one line
- Remove !important from css  for best practices
- Validate code in W3C
**New Changes After Review**
- Made sure that the navbar links is taking  in one line
- Used the recommended stickler configuration
- Removed commented code in css.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+}
+
+a {
+  text-decoration: none;
+}
+
+/* HEADER */
+
+.header-wrapper {
+  background-color: #f72210 !important;
```
Comment (andresporras) [inline css/style.css]: It's not a good practice to use !important

File: index.html [hunk 2]
```diff
@@ -0,0 +1,2466 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (andresporras) [inline index.html]: Replace this line with:
`<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,2466 @@
...
+						<li class="nav-item">
+							<a class="nav-link pt-0 border-right font-weight-bold"
+								>Business</a
+							>
+						</li>
+						<li class="nav-item">
+							<a class="nav-link pt-0 border-right font-weight-bold"
+								>Tech &amp; Science</a
+							>
+						</li>
```
Comment (andresporras) [inline index.html]: Make sure the text of your links always stays in one line (like in the original page)

**Comment by andresporras** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

**Review by andresporras — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle:
**Hello @uimarshall** 
Awesome job, you make great use of bootstrap classes, your page looks really close to the original  :clap:
There are still some improvements to do:

- [ ]  Check HTML markup with this validator https://validator.w3.org/#validate_by_upload By now the validator shows some errors in your markup.
- [ ] Don't use !important in your CSS
- [ ] Make sure the text of navbar links stay always in one line, like in the original page

Check the inline comments I did in some parts of your code 

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,507 @@
...
+.newsletter-form {
+  background-color: #f4f5f6;
+}
+
+.newsletter-form img,
+.in-the-magazine-header img {
+  width: 29px;
+  height: 28px;
+
+  /* margin-right: 10px; */
```
Comment (HADDADSOHAIB) [inline css/style.css]: This should be removed

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @uimarshall    :handshake: 

### Good job so far :clap:  your project is looking good :medal_sports: 

### there are some changes to get the project approved :point_down:
- The nav bar still taking two lines:
Your project:
Screenshot from 2020-06-05 11-02-02
Original:
Screenshot from 2020-06-05 11-02-18
Please try to fix that, it should look like the original.
- I think you modified your stickler configuration, please use this file and don't change it:
https://github.com/microverseinc/linters-config/blob/Stickler/css/stylelint.config.js
- There commented code in your CSS, please remove that:
- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
### After you are done with the changes, ask for another code review, Good Luck :+1:

**Comment by uimarshall** [comment]
> > ### Pull Request under the review by Sohaib Haddad. (TSE)
> 
> ### Change Required ♻️
> ### Hello @uimarshall 🤝
> ### Good job so far 👏 your project is looking good 🏅
> ### there are some changes to get the project approved 👇
> * The nav bar still taking two lines:
>   Your project:
>   Screenshot from 2020-06-05 11-02-02
>   Original:
>   Screenshot from 2020-06-05 11-02-18
>   Please try to fix that, it should look like the original.
> * I think you modified your stickler configuration, please use this file and don't change it:
>   https://github.com/microverseinc/linters-config/blob/Stickler/css/stylelint.config.js
> * There commented code in your CSS, please remove that:
> * [ ]  Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
> 
> ### After you are done with the changes, ask for another code review, Good Luck 👍
@HADDADSOHAIB I did fix the navbar, I think I did not update it on the live demo.
I will update now.

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

**Review by andresporras3423 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello @uimarshall** 
Amazing job :clap: :clap: you have met all the requirements for this project
Now you are ready to continue with your next challenge :muscle: 
Don't forget to merge with the master branch

Keep going like this,

Oscar Andrés Russi Porras

**Pull request merged**